src/features/venue/components/VenueOffers/VenueOffers.native.test.tsx

```diff
@@ -253,5 +253,52 @@ describe('<VenueOffers />', () => {
 
       expect(screen.queryByTestId('allGtlPlaylistsTitle')).not.toBeOnTheScreen()
     })
+
+    it('Analytics - should log ConsultOffer that user opened the offer', async () => {
+      render(reactQueryProviderHOC(<VenueOffers venueId={mockVenue.id} playlists={playlists} />))
+
+      const tileImageButton = await screen.findAllByTestId('tileImage')
+      fireEvent.press(tileImageButton[0])
+
+      expect(analytics.logConsultOffer).toHaveBeenNthCalledWith(1, {
+        offerId: Number(VenueOffersResponseSnap[1].objectID),
+        from: 'venue',
+        venueId,
+      })
+    })
+
+    it('should prepopulate react-query cache when clicking on offer', async () => {
+      render(reactQueryProviderHOC(<VenueOffers venueId={mockVenue.id} playlists={playlists} />))
+
+      const tileImageButton = await screen.findAllByTestId('tileImage')
+      fireEvent.press(tileImageButton[0])
+
+      const queryHash = JSON.stringify(['offer', Number(VenueOffersResponseSnap[1].objectID)])
+      const query = queryCache.get(queryHash)
+
+      expect(query).not.toBeUndefined()
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      expect(query!.state.data).toStrictEqual({
```

Comment:
dans un autre test, il y a ça

```suggestion
      expect(query?.state.data).toStrictEqual({
```

si c'est suffisant, ça me semble mieux, car on évite le disable next line